src/main/resources/templates/myProfile.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <title>My Profile</title>
+</head>
+<body>
+    <div id="nav-placeholder"></div><br><br>
+    <form action="#" th:action="@{/myProfile}" th:object="${myProfile}"  method="post"
+          class="container d-flex flex-column flex-md-col justify-content-between col-sm-12 col-md-6 col-lg-4">
+
+        <div class="mx-auto" style="height: auto ; width: auto">
+            <h3>My Profile</h3><br>
+            <div th:object="${myProfile}">
+                <span>User Name:</span>
+                <span class="text-blue" th:text="*{userName}"></span><br>
+                <span>Email:</span>
+                <span class="text-blue" th:text="*{email}"></span><br>
+<!--<button class="btn btn-primary" >Edit your Profile </button><br><br> this should be a button wich opens a new page,
```

Comment:
ok. thank you for the comment but it should be waaaay more general! do we want this button? if so then we should find a way to implement it. if there is a plan to implement just write something like "Button should be implemented in issue ..." and add the number of the issue. or just leave it our completely